include/alpaka/event/EventGenericSycl.hpp

```diff
@@ -45,7 +45,14 @@ namespace alpaka::experimental
             return !(lhs == rhs);
         }
 
+        [[nodiscard]] auto getNativeHandle() const noexcept
```

Comment:
```suggestion
        [[nodiscard]] auto getNativeHandle() const
```

`sycl::event`'s copy constructor is not `noexcept`.